src/misc/ios/SDL_sysurl.m

```diff
@@ -35,7 +35,15 @@ bool SDL_SYS_OpenURL(const char *url)
 #else
         NSString *nsstr = [NSString stringWithUTF8String:url];
         NSURL *nsurl = [NSURL URLWithString:nsstr];
-        return [[UIApplication sharedApplication] openURL:nsurl];
+        if (![[UIApplication sharedApplication] canOpenURL:nsurl]) {
+            return SDL_SetError("No handler registerd for this type of URL");
+        }
+        if (@available(iOS 10.0, *)) {
```

Comment:
You need a tvOS availability here.

Reply:
Whoops, I thought the `#ifdef` handled this, but it's for VisionOS...how the heck does VisionOS not let you pop up a web browser window? Even Steam can do that in a fullscreen game!  lol

Reply:
Oh, VisionOS can, it just removed the deprecated API from the start. It has the newer one.

Reply:
Pushed a fix in 3bd759e420df3ec7acb2bb5ea37ca1ede9c2f6be, but I have no idea if this is right.